Makefile

```
@@ -70,6 +71,10 @@ release:
 	    -a \
 	    -ldflags "-w -X main.Version=$(VERSION)" \
 	    ./cmd/compound
+	@CGO_ENABLED=0 go build -o $(EXECUTABLE2) \
+	    -a \
+	    -ldflags "-w -X main.Version=$(VERSION)" \
+	    ./cmd/dnsman2
```

nit: Similar to above, maybe we can go for a unified naming approach of using `ng/next-generation` wherever possible (with acceptable effort)?
```suggestion
	@CGO_ENABLED=0 go build -o $(EXECUTABLE_NG) \
	    -a \
	    -ldflags "-w -X main.Version=$(VERSION)" \
	    ./cmd/dnsman-ng
```